index.js

```diff
@@ -177,13 +206,13 @@ handlers[Dota2.EGCBaseClientMsg.k_EMsgGCClientConnectionStatus] = function gcCli
 
 Dota2.Dota2Client = Dota2Client;
 
-require("./handlers/cache");
+//require("./handlers/cache");
```

Comment:
I think those two are still broken, at least that's what i gather from
their contents. I haven't done any debugging on things that didn't work
before/were only partially implemented. As for the other handler, I'm still
testing them as i go, for the moment I'm pretty sure lobbies and chat are
ok: I'm able to create a lobby, detect changes, start it, ... for chat i'm
able to retrieve the list of open channels. In the following days i'll
probably do some more test, but i wanted at least to already make the pull
request so we're not all doing the same work.

On 20 August 2015 at 10:53, Howard Chung <notifications@github.com> wrote:

> In index.js
> <https://github.com/RJacksonm1/node-dota2/pull/90#discussion_r37507774>:
>
> > @@ -177,13 +206,13 @@ handlers[Dota2.EGCBaseClientMsg.k_EMsgGCClientConnectionStatus] = function gcCli
> >
> >  Dota2.Dota2Client = Dota2Client;
> >
> > -require("./handlers/cache");
> > +//require("./handlers/cache");
>
> The commented-out handlers probably need to be enabled if you've tested
> that they work.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RJacksonm1/node-dota2/pull/90/files#r37507774>.
>